### PR TITLE
fixed the error occured due to appending of the stars to the wallet

### PR DESCRIPTION
**Pull request by UmairMukhtar**

### CODE REVIEW

**Comment by UmairMukhtar** [comment]
Screenshot from 2023-02-16 20-45-50
Screenshot from 2023-02-16 20-45-58
Screenshot from 2023-02-16 20-46-07
Screenshot from 2023-02-16 20-47-06

**Comment by UmairMukhtar** [comment]
@AltiMario i tried with a different user , correct user is being linked with the creation
Screenshot from 2023-02-16 22-08-59

**Comment by UmairMukhtar** [comment]
also i have tried with save as draft functionality mentioned here https://github.com/eLearningDAO/POCRE/issues/282 it is working fine in this pull request

File: app/web-frontend/src/pages/Creations/common/form/Steps/Two.jsx [hunk 1]
```diff
@@ -334,10 +334,18 @@ export default function StepTwo({
...
+    const authorWithoutStars = [];
+    updatedValues.author.map(
+      (author) => {
+        const nameWithouStars = author.split('-')[0];
+        authorWithoutStars.push(nameWithouStars);
+        return true;
+      },
+    );
+    updatedValues.author = authorWithoutStars;
+    setMaterials([...materials, updatedValues]);
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/web-frontend/src/pages/Creations/common/form/Steps/Two.jsx]: done

File: app/web-frontend/src/pages/Creations/common/form/Steps/Two.jsx [hunk 2]
```diff
@@ -309,7 +309,7 @@ function AuthorInviteModal({ onClose = () => {}, onAuthorDetailsSubmit }) {
 }
 
 const getAuthor = (author, user) => {
-  let authorName = `${author.user_name}${user.user_id === author?.user_id ? ' (You) ' : ' '}`;
+  let authorName = `${author.user_name}${user.user_id === author?.user_id ? ' (You)-' : '-'}`;
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/web-frontend/src/pages/Creations/common/form/Steps/Two.jsx]: fixed

**Comment by UmairMukhtar** [comment]
After Moving the code the stars logic to useCreationForm

Screenshot from 2023-02-17 21-55-46
Screenshot from 2023-02-17 21-56-21
Screenshot from 2023-02-17 21-57-21
Screenshot from 2023-02-17 21-57-40

**Review by AltiMario — APPROVED** [review]

**Pull request merged**